projects/distributed-tracing/src/shared/dashboard/widgets/waterfall/waterfall/waterfall-chart.ts

```diff
@@ -37,3 +38,12 @@ export interface WaterfallChartState {
   children: WaterfallDataNode[];
   expanded: boolean;
 }
+
+export interface LogEvent {
+  [key: string]: unknown;
```

Comment:
what unknown fields does this have? would anything break if you removed this?

Reply:
Actually I added this as I was using this as a table row, but lately we did not use that so now removed it.
Fixed!!